airbyte-integrations/connector-templates/java-destination/Destination.java.hbs

```diff
@@ -27,9 +27,9 @@ package io.airbyte.integrations.destination.{{snakeCase name}};
 import com.fasterxml.jackson.databind.JsonNode;
 import io.airbyte.commons.json.Jsons;
 import io.airbyte.commons.resources.MoreResources;
-import io.airbyte.integrations.base.Destination;
-import io.airbyte.integrations.base.DestinationConsumer;
-import io.airbyte.integrations.base.IntegrationRunner;
+import Destination;
```

Comment:
Is that correct?

Reply:
oops. nope. intellij didn't get that one right. i'll go through the template more carefully.